apps/challenges/views.py

```diff
@@ -3026,16 +3036,28 @@ def scale_resources_by_challenge_pk(request, challenge_pk):
     ):
         response = scale_resources(challenge, worker_cpu_cores, worker_memory)
         if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
-            response_data = {
-                "error": "Issue with ECS."
-            }
+            if response.get('Error', {'Message': 'No error', 'Code': 'No error'}).get('Code', 'No error code') == \
+                    'ClientException':
+                response_data = {
+                    "error": "Challenge workers are inactive or do not exist."
+                }
```

Comment:
This case won't occur because you are checking `if worker is None` on L3023. We can remove this error response and just send a error message saying `"error": "Scaling worker resources failed!"`

Reply:
I added this case because while testing on staging, the recorded and actual worker counts often became out of sync, leading to the response not matching the actual situation. Doing it like this should help debug these desyncs and fix the cause of these issues.